core/README.md

```diff
@@ -0,0 +1,49 @@
+## :core
+
+This is the core of the library, published as the `dataframe-core` package.
+It contains the DataFrame API and its implementation, as well as plenty of JUnit tests.
+
+I/O operations are split off into other modules, like [:dataframe-excel](../dataframe-excel),
+or [:dataframe-jdbc](../dataframe-jdbc), however, this is has not happened yet for all operations
+(see [Issue #100](https://github.com/Kotlin/dataframe/issues/100)).
+
+Operations likely to be split off in the future include:
```

Comment:
probably better to remove about future and fix the current state and say, that these data sources are part of Core